shell/pages/c/_cluster/_product/projectsnamespaces.vue

```diff
@@ -325,7 +333,7 @@ export default {
           </div>
           <div class="right">
             <n-link
-              v-if="isNamespaceCreatable"
+              v-if="isNamespaceCreatable && (canSeeProjectlessNamespaces || group.group.key !== notInProjectKey)"

```

Comment:
This is how i originally implemented the fix, but thought the change to `showMockNotInProjectGroup` covered all cases. Good catch!